vectordatasource/meta/function.py

```diff
@@ -407,13 +407,24 @@ def tz_estimate_parking_capacity(capacity, parking, levels, way_area):
     try:
         # if the tags tell us what capacity is, then we should respect that.
         capacity = int(capacity)
+
+        # don't allow abnormally large capacity values
+        # (West Edmonton mall has 20k spaces)
+        if capacity > 99999:
+            return None
+
         return capacity
 
     except (ValueError, TypeError):
         # sometimes people don't put integers in the capacity, which is kind of
         # annoying. it means we just have to fall back to estimating.
         pass
 
+    # don't try to do this if way_area is abnormally large
+    # (Epcot's parking lot is 647k m^2)
+    if way_area > 2000000:
```

Comment:
Is it beneficial to print some log for these conditions? I don't see any log in this file, do we log in the caller?